pkg/k8s/apps/crud.go

```diff
@@ -165,3 +165,26 @@ func TranslateDevMode(trMap map[string]*Translation) error {
 	}
 	return nil
 }
+
+// ListDevModeOn returns a list of strings with the names of deployments or statefulsets in DevMode
+func ListDevModeOn(ctx context.Context, namespace string, c kubernetes.Interface) ([]string, error) {
+	devModeApps := make([]string, 0)
+
+	deploymentsDev, err := deployments.List(ctx, namespace, model.DevLabel, c)
```

Comment:
Shouldn't the label selector be `fmt.Sprintf("%s=true", model.DevLabel)`? Same for statefulsets

Reply:
I tested it and when the component was in dev mode the label appeared alone without "true" or "false", but i guess is better to actively look for the true ones 👍🏻